mozdownload/scraper.py

```diff
@@ -51,6 +51,9 @@
                            'win32': 'exe',
                            'win64': 'exe'}
 
+DEFAULT_BRANCHES = {'thunderbird': 'comm-central'
```

Comment:
No, `b2g` is dead, and we should remove it from this repository. I will file a separate issue for this. Thanks for noticing!